server/handlers/order.js

```diff
@@ -0,0 +1,51 @@
+var dao = require("../dao/dao");
+const { Order } = require("../models/order");
+const { validationResult } = require("express-validator");
+const {
+  clientIDPathValidator,
+  productIDPathValidator,
+  productQtyPathValidator,
+  productsValidator,
+} = require("./shared_validators");
+
+exports.createOrderValidatorChain = [
+  clientIDPathValidator,
+  productsValidator,
+  productIDPathValidator,
+  productQtyPathValidator,
+];
+
+exports.createOrderHandler = async function (req, res, next) {
+  // Insert the new employee
+  var result;
+  var totalPrice = 0;
```

Comment:
Add a comment to specify that 'productPrice' is hardcoded to 1 as a temporary solution